### DISCUSSION TITLE
Bulk delete objects will cause CPU high?

### DISCUSSION THREAD

**Discussion by sophiah** [minio/minio#18332] [Q&A] [answered]
Hi, 

I wonder know if I invoked hugh remove objects requests, will it cause the cluster CPU loading high? If yes, how do I resolved this CPU loading issues?

**Comment by klauspost** [comment, marked as answer]
Bulk operations are the same as potential thousands of individual requests - so load will be equivalent to that.

We do not perform community performance analysis, so we can't help you much further.

**Reply by sophiah** [reply]
I see thank you anyway
